src/contribute-to-tidb/write-document.md

```diff
@@ -37,51 +37,33 @@ Before you contribute, let's take a quick look at some general information about
-| `release-5.2` branch | the 5.2 version |
-| `release-5.1` branch | the 5.1 version |
-| `release-5.0` branch | the 5.0 version |
-| `release-4.0` branch | the 4.0 version |
-| `release-3.1` branch | the 3.1 version |
-| `release-3.0` branch | the 3.0 version |
-| `release-2.1` branch | the 2.1 version |
-
-Each docs version is updated very frequently and changes to one version often apply to another version or other versions as well. We introduce ti-chi-bot to automatically file PRs to other versions as long as you add corresponding cherry-pick labels to your PR.
+- The [documentation under development](https://docs.pingcap.com/tidb/dev) is maintained in the `master` branch.
+- The [published documentation](https://docs.pingcap.com/tidb/stable/) is maintained in the corresponding `release-<verion>` branch.
+- The [archived documentation](https://docs-archive.pingcap.com/) is no longer maintained and does not receive any further updates.
 
 ### Use cherry-pick labels
 
-- If your changes apply to only one doc version, just submit a PR to the corresponding version branch.
+As changes to one doc version often apply to other doc versions as well, we introduce ti-chi-bot to automate the PR cherry-pick process based on cherry-pick labels.
+
+- If your changes apply to only one doc version, just create a PR based on the branch of the doc version. There is no need to add any cherry-pick labels.
 
-- If your changes apply to multiple doc versions, you don't have to submit a PR to each branch. Instead, after you submit your PR, trigger the ti-chi-bot to submit a PR to other version branches by adding one or several of the following labels as needed. Once the current PR is merged, ti-chi-bot will start to work.
-    - `needs-cherry-pick-release-5.2` label: ti-chi-bot will submit a PR to the `release-5.2` branch.
-    - `needs-cherry-pick-release-5.1` label: ti-chi-bot will submit a PR to the `release-5.1` branch.
-    - `needs-cherry-pick-release-5.0` label: ti-chi-bot will submit a PR to the `release-5.0` branch.
-    - `needs-cherry-pick-release-4.0` label: ti-chi-bot will submit a PR to the `release-4.0` branch.
-    - `needs-cherry-pick-release-3.1` label: ti-chi-bot will submit a PR to the `release-3.1` branch.
-    - `needs-cherry-pick-release-3.0` label: ti-chi-bot will submit a PR to the `release-3.0` branch.
-    - `needs-cherry-pick-release-2.1` label: ti-chi-bot will submit a PR to the `release-2.1` branch.
-    - `needs-cherry-pick-master` label: ti-chi-bot will submit a PR to the `master` branch.
+- If your changes apply to multiple doc versions, instead of creating multiple PRs, you can just create one PR based on the latest applicable branch, and then add one or several `needs-cherry-pick-release-<version>` labels to the PR according to the applicable doc versions. Then, after the PR is merged, ti-chi-bot will automatically create the corresponding cherry-pick PRs based the branches of the specified branches.
```

Comment:
```suggestion
- If your changes apply to multiple doc versions, instead of creating multiple PRs, you can just create one PR based on the latest applicable branch, and then add one or several `needs-cherry-pick-release-<version>` labels to the PR according to the applicable doc versions. Then, after the PR is merged, ti-chi-bot will automatically create the corresponding cherry-pick PRs based on the branches of the specified versions.
```